account.go

```diff
@@ -111,6 +112,33 @@ type Account struct {
 	Deleted        bool     `json:"deleted"`
 }
 
+type ExternalAccounts struct {
+	Object     string            `json:"object"`
+	Data       []ExternalAccount `json:"data"`
+	HasMore    bool              `json:"has_more"`
+	TotalCount uint              `json:"total_count"`
+	Url        string            `json:"url"`
+}
+
+type ExternalAccount struct {
+	BankAccount *BankAccount
+	Card        *Card
+}
+
+func (ea *ExternalAccount) UnmarshalJSON(b []byte) (err error) {
+	bank, card := BankAccount{}, Card{}
+	if err = json.Unmarshal(b, &bank); err == nil {
```

Comment:
Would this actually return an error? It might just fill the field's that it's able to and return a card as a bank account. We'll certainly need to add some tests to verify that one way or another :$

Unfortunately, we may need to do an initial deserialization pass to a `map[string]interface{}` and then check its object property before deserializing it again appropriately.